src/store/slices/blogsSlice.js

```diff
@@ -2,40 +2,21 @@ import { createSlice } from "@reduxjs/toolkit";
 
 const blogsSlice = createSlice({
     name: 'blogs',
-    initialState : [
```

Comment:
This was not looking good from start, I didn't know whats wrong but something wasn't clicking for me :). Nice change